Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM busybox as test
+
+ADD docker-installer/install_cni_plugins.sh /script/install_cni_plugins.sh
+ADD docker-installer/test_install_cni_plugins.sh /script/test_install_cni_plugins.sh
+WORKDIR /script
+RUN /script/test_install_cni_plugins.sh
+
+FROM busybox as build
+ARG TAG
+# Get buildx automatic platform vars: https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope
+ARG TARGETPLATFORM
+ARG TARGETOS
+ARG TARGETARCH
+ARG TARGETVARIANT
+ARG BUILDPLATFORM
+ARG BUILDOS
+ARG BUILDARCH
+ARG BUILDVARIANT
+RUN echo TARGETPLATFORM=$TARGETPLATFORM
+RUN echo TARGETOS=$TARGETOS
+RUN echo TARGETARCH=$TARGETARCH
+RUN echo TARGETVARIANT=$TARGETVARIANT
+RUN echo BUILDPLATFORM=$BUILDPLATFORM
+RUN echo BUILDOS=$BUILDOS
+RUN echo BUILDARCH=$BUILDARCH
+RUN echo BUILDVARIANT=$BUILDVARIANT
+# Use buildx automatic platform vars
+COPY release-$TAG/cni-plugins-$TARGETOS-$TARGETARCH-$TAG.tgz cni-plugins-$TARGETOS-$TARGETARCH-$TAG.tgz
+COPY release-$TAG/cni-plugins-$TARGETOS-$TARGETARCH-$TAG.tgz.sha512 cni-plugins-$TARGETOS-$TARGETARCH-$TAG.tgz.sha512
+RUN set -eux; \
+    sha512sum -c cni-plugins-$TARGETOS-$TARGETARCH-$TAG.tgz.sha512; \
+    mkdir -p /opt/cni/bin; \
+    tar -xvf cni-plugins-$TARGETOS-$TARGETARCH-$TAG.tgz -C /opt/cni/bin;
+
+# This is the final, minimal container
+FROM busybox as final
+COPY docker-installer/install_cni_plugins.sh /script/install_cni_plugins.sh
```

Comment:
You may need `LABEL` command to specify some of label, as [github document](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#labelling-container-images) mentioned, such as `org.opencontainers.image.source`

As I worked similar stuff in another repo, this org.opencontainers.image.source is required.

Reply:
Thanks for the tip. Any recommendation on which `LABEL`s to add?

Reply:
managed to add all the `org.opencontainers.image.xxx` labels, hope i didn't miss anything.